pawno/include/PPC_Defines.inc

```diff
@@ -1016,27 +1016,27 @@ new ACommands[][TCommands] =
 	{1, "/jail <PlayerToJail> <JailTime> <Reason>",                     "Jails another player for a certain time in seconds"},
 	{1, "/kick <PlayerToKick> <Reason>",                                "Kick a player"},
 	{1, "/mute <OtherPlayer> <Reason>",                                 "Mute a player"},
-	{1, "/mutes",                                   					"Display muted players"},
+	{1, "/muted",                                   					"Display muted players"},
 	{1, "/nos",                                   						"Adds nitro to your vehicle"},
 	{1, "/plane",                                   					"Spawns a plane of your choice"},
 	{1, "/port <OtherPlayer>",                                   		"Teleport yourself to another player"},
 	{1, "/portloc <x> <y> <z>",                                   		"Teleport yourself to the given coordinates"},
 	{1, "/portvehicle <CarID>",                                   		"Teleport yourself to a vehicle"},
 	{1, "/rep",                                   						"Show the last 50 reports in a dialog"},
 	{1, "/repair",                                   					"Repairs your vehicle for free"},
-	{1, "/respawn <Vehicle-id>",                                   		"Forces a vehicle to respawn"},
+	{1, "/respawnvehicle <Vehicle-id>",                                   		"Forces a vehicle to respawn"},
 	{1, "/spec <OtherPlayer>",                                   		"Spectate a player"},
 	{1, "/tele <PlayerToPort> <TargetPlayer>",                          "Teleport a player to another player"},
 	{1, "/trailer",                                   					"Spawns a trailer of your choice"},
 	{1, "/unmute <OtherPlayer>",                                   		"Un-mutes a player"},
 	{1, "/warn <PlayerToWarn> <Reason>",                                "Warn a player, up to three times"},
 	{2, "/announce <Message> <Style (optional)>",                  		"Shows a message on player's screen (GameText)"},
 	{2, "/eject <OtherPlayer>",                                   		"Ejects a player from his vehicle"},
-	{2, "/wanted <PlayerID> <Stars>",                                   "Set a player's wanted level"},
+	{2, "/setwanted <PlayerID> <Stars>",                                   "Set a player's wanted level"},
 	{3, "/ban <PlayerToBan> <Days> <Hours> <Reason>",                   "Ban a player for a certain time"},
 	{3, "/caroption",                                   				"Changes some options for your vehicle"},
-	{3, "/cleanupcar <VehicleID>",                                   	"Deletes a vehicle spawned by an admin"},
-	{3, "/cleanupcars",                                   				"Removes all spawned vehicles from the map"},
+	{3, "/cleanupvehicle <VehicleID>",                                   	"Deletes a vehicle spawned by an admin"},
+	{3, "/cleanupvehicles",                                   				"Removes all spawned vehicles from the map"},
```

Comment:
We need to set this to `/cleanupallvehicles` to avoid admins add a "s" by mistake. This would remove all vehicles. So `/cleanupallvehicles` is safer.